ws-server/src/server.rs

```diff
@@ -324,7 +327,8 @@ async fn background_task(
 
 		// Force `conn_tx` to this async block and close it down
 		// when the connection closes to be on safe side.
-		conn_tx.close();
+		// conn_tx.close();
+		close_notify_server_stop.notify_one();
```

Comment:
@niklasad1 I don't understand the comment here fully and I'm not sure if this is actually needed anymore. Should I keep this?

Reply:
We want to ensure that `close_notify` gets dropped/notified when connection task is terminated.

So I think what you did is correct